test/test-fs-event.c

```diff
@@ -467,6 +488,43 @@ TEST_IMPL(fs_event_watch_file) {
   return 0;
 }
 
+TEST_IMPL(fs_event_watch_file_exact_path) {
+  /*
+    This test watches a file named "file.jsx" and modifies a file named
+    "file.js". The test verifies that no events occur for file.jsx.
+  */
+  uv_loop_t* loop = uv_default_loop();
```

Comment:
declare and assign in different lines or msvc 2012 will complain here (or move the definition of r above)